packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -13,6 +13,9 @@ export const iconMetadata: {
     IconAdd: {
         tags: ['add input field']
     },
+    IconAiChatBot: {
```

Comment:
I see `message-bot` on font awesome but not "ai chat bot". The font awesome name `message-bot` also fits better with the terminology we are using in our elements: https://fontawesome.com/search?q=ai%20chat&o=r